mars/oscar/core.pyx

```diff
@@ -43,6 +46,54 @@ def set_debug_options(options):
         _log_cycle_send = options.log_cycle_send
 
 
+cdef _get_local_actor(address, uid):
+    # Do not expose this method to Python to avoid actor being
+    # referenced everywhere.
+    #
+    # The cycle send detection relies on send message, so we
+    # disabled the local actor proxy if the debug option is on.
+    if _log_cycle_send:
+        return None
+    pool = _local_pool_map.get(address)
+    if pool is not None:
+        actor = pool._actors.get(uid)
+        if actor is not None:
+            return actor
+    return None
+
+
+cdef class ActorPool:
```

Comment:
I can change the code to a pool registration function, but there is no corresponding unregistration function because I use a weak value dict to track the pool instances.

Reply:
I changed the `ActorPool` base to a `register_local_pool` function.